jobs/pipelines/powervs/poll-powervs-images/Jenkinsfile

```diff
@@ -18,8 +20,9 @@ pipeline {
     }
     environment {
         //users and credentials. All must be defined in Jenkins Credentials
-        IBMCLOUD_API_KEY = credentials('IBMCLOUD_UPSTREAM_CI_API_KEY')
-        PVSADM_VERSION = "v0.1.5"
+        IBM_CLOUD_API_KEY = credentials('IBMCLOUD_API_KEY')
```

Comment:
Please change IBMCLOUD_API_KEY to IBMCLOUD_UPSTREAM_CI_API_KEY

Reply:
I have changed it